web/src/components/Accordion.tsx

```diff
@@ -0,0 +1,67 @@
+import { useState } from 'react';
+import { HiChevronDown, HiChevronUp } from 'react-icons/hi2';
+import { twMerge } from 'tailwind-merge';
+
+export default function Accordion({
+  isCollapsedDefault = true,
+  onClick,
+  badge,
+  className,
+  icon,
+  children,
+  title,
+}: {
```

Comment:
This should probably also accept a atom so we can perserve the state in a way where the component is still reusable.

Reply:
What state do we need to preserve? If the accordion is collapsed or expanded?

Reply:
Yeah so if you switch between zones it stays open and don't auto close again. And by passing a atom we can do this for each graph later on.

Reply:
I have made it accept an atom now. I also made an atom for datasources and I don't know if we actually want this. What do you think?